sample_keyprovider/src/enc_mods/crypto.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use aes_gcm::{aead::Aead, aes::Aes256, Aes256Gcm, Key, Nonce};
+use anyhow::*;
+use strum_macros::EnumString;
+
+pub const HARDCODED_KEY: &[u8] = &[
```

Comment:
Maybe add a comment saying that this is just for the sample.

Reply:
Great, I'll add that